pandas-iris/{{ cookiecutter.repo_name }}/src/{{ cookiecutter.python_package }}/pipelines/data_science/README.md

```diff
@@ -1,58 +1,56 @@
-# Data Science pipeline
+# Pipeline data_science
 
-> *Note:* This `README.md` was generated using `Kedro {{ cookiecutter.kedro_version }}` for illustration purposes. Please modify it according to your pipeline structure and contents.
+> *Note:* This is a `README.md` boilerplate generated using `Kedro {{ cookiecutter.kedro_version }}`.
 
 ## Overview
 
 This modular pipeline:
-1. trains a simple multi-class logistic regression model (`train_model` node)
-2. makes predictions given a trained model from (1) and a test set (`predict` node)
-3. reports the model accuracy on a test set (`report_accuracy` node)
-
+1. splits the data into training dataset and testing dataset using a configurable ratio found in `conf/base/parameters/data_science.yml`
+2. trains a simple 1-nearest neighbour model (`train` node) and makes prediction dataset.
```

Comment:
I like the example is simpler now. It may be just wordings, but KNN is a special algorithm that does not require `training`.

Reply:
Maybe `runs a simple 1-nearest neighbour model` instead?